app/assets/javascripts/initializers/initializeHeroBannerClose.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+function initializeHeroBannerClose() {
+  let banner = document.getElementById("js-she-coded-hero-banner");
```

Comment:
I noticed that the element id is tied to our current campaign, so we'll need to change this code for another campaign.
What do you think about changing the name to a more universal one?